src/app/my-settings/components/my-settings-notifications/my-settings-notifications.component.html

```diff
@@ -0,0 +1,75 @@
+<div
+  class="erz-container erz-container-limited erz-container-padding-y erz-container-padding-x-responsive"
+  *erzLet="{
+    notificationFormGroup: notificationFormGroup$ | async
+  } as data"
+>
+  <!-- setting-header -->
+  <h5 class="m-0" class="setting-header">
+    {{ 'my-settings.notifications.title' | translate }}
+  </h5>
+
+  <form
+    *ngIf="data.notificationFormGroup"
+    [formGroup]="data.notificationFormGroup"
+    (ngSubmit)="onSubmit()"
+  >
+    <!-- setting-entry -->
+    <div class="checkbox-setting">
+      <input
+        id="my-settings-notifications-gui"
+        type="checkbox"
+        class="checkbox-setting-input"
+        formControlName="notificationsGui"
+      />
+      <label class="checkbox-setting-label" id="my-settings-notifications-gui">
```

Comment:
Bei den Labels sollte es statt "id" "for" sein - dass ist auch gleich das mit dem Klicktarget richtig.

Reply:
Hab ich angepasst.